modules/issue_tracker/jsx/issueTrackerIndex.js

```diff
@@ -263,14 +276,37 @@ class IssueTrackerIndex extends Component {
     ];
 
     return (
-      <FilterableDataTable
-        name="issuesTracker"
-        data={this.state.data.data}
-        fields={fields}
-        filterPresets={filterPresets}
-        actions={actions}
-        getFormattedCell={this.formatColumn}
+      <div>
+      <div className="view-toggle">
```

Comment:
add condition on this Div -- for #9422, like how permissions are used to show/hide buttons [in other modules](https://github.com/aces/Loris/blob/32377a205d168683b6542323ca0fd4b99b8aeda7/modules/candidate_parameters/jsx/FamilyInfo.js#L132)